config/maps.txt

```diff
@@ -68,7 +68,13 @@ endmap
 map PubbyStation
 	votable
 	maxplayers 30
-	default
+	#default
```

Comment:
Leave the map config edit for now during the test merge phase. easier to handle that adhoc on the box